<?xml version="1.0" encoding="UTF-8"?>
<!--Copyright 2024 The MathWorks, Inc.-->

<rsccat xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0" locale="zh_CN" product="parallel" xsi:noNamespaceSchemaLocation="../../resources/schema/msgcat.xsd">
  <message>
    <entry key="AttachedFilesLink" context="uistring">列出文件</entry>
    <entry key="AttachedFilesList" context="uistring">并行池的附加文件:</entry>
    <entry key="AttachedFilesTransferError" context="error">Failed to transfer files to the parallel pool workers.\nThis was caused by: {0}</entry>
    <entry key="AutoAttachedFiles" context="uistring">自动添加到并行池的文件:\n{0}</entry>
    <entry key="CannotDisableSPMD" context="warning">此集群类型不支持在并行池上禁用 SPMD。请将 'SpmdEnabled' 值设置为 true。</entry>
    <entry key="CannotSetAttachedFilesProperty" context="error">To attach files to a parallel pool use addAttachedFiles(pool, files), where files is a cell array of filenames.</entry>
    <entry key="DeletedObject" context="uistring">删除的对象。</entry>
    <entry key="DiaryTruncated" context="warning">日志输出被截断。</entry>
    <entry key="DisallowedParallelFeature" context="error">Use of {0} is not supported in a thread-based parallel pool. Choose another parallel environment, for example 'Processes'.</entry>
    <entry key="DoneDependencyAnalysis" context="uistring">完成。</entry>
    <entry key="EmptyClusterValue" context="error">Need to provide a ''parallel.Cluster'' object returned by the function ''parcluster'' in a name-value pair, where name is ''Cluster''.</entry>
    <entry key="FailedToInitializeInteractiveSession" context="error">Parallel pool failed to start.</entry>
    <entry key="FailedToRestart" context="warning">无法重新启动并行池。原因: {0}</entry>
    <entry key="FailedToStartSessionServices" context="error">Error starting pool services:\n{0}</entry>
    <entry key="FevalQueueManyRequests" context="parameter">{0,number,integer} 个未决 future。</entry>
    <entry key="FevalQueueOneRequest" context="parameter">1 个未决 future。</entry>
    <entry key="FileTransferError" context="error">Error detected while transferring files to workers; try again.</entry>
    <entry key="FileTransferObserverError" context="error">Unexpected error in FileTransferObserver:\n{0}</entry>
    <entry key="IdleTimeoutFiniteManyMinutes" context="parameter">{0,number,integer} 分钟</entry>
    <entry key="IdleTimeoutInfinite" context="parameter">Inf (无自动关闭)</entry>
    <entry key="IdleTimeoutOneMinute" context="parameter">1 分钟</entry>
    <entry key="IgnoringAlreadyAttachedFiles" context="warning">已附加的文件将被忽略。要查看附加了哪些文件，请查看并行池的 ''AttachedFiles'' 属性。</entry>
    <entry key="InternalRuntime" context="error">An internal runtime error occurred.</entry>
    <entry key="InternalRuntimeWithCause" context="error">An internal runtime error occurred.\nThis was caused by: {0}.</entry>
    <entry key="InvalidDeleteSyntax" context="error">Invalid parallel pool deletion syntax. Use delete(pool) instead.</entry>
    <entry key="InvalidDependency" context="error">{0} is not a valid file or directory.</entry>
    <entry key="InvalidIdleTimeout" context="error">The IdleTimeout parameter must be a scalar integer greater than zero.</entry>
    <entry key="InvalidLogicalParameter" context="error">Value must be logical ''true'' or ''false''.</entry>
    <entry key="InvalidSingleInput" context="error">Invalid input for parpool. Allowed arguments are a profile name, a cluster object, or an integer specifying the pool size.</entry>
    <entry key="InvalidTwoInputFirstArgument" context="error">Invalid input for parpool. Allowed value for the first argument is a profile name or a cluster object.</entry>
    <entry key="InvalidTwoInputSecondArgument" context="error">parpool 的输入无效。第二个参数的允许值是指定池大小的整数。</entry>
    <entry key="LostAnUnknownWorker" context="error">The client lost connection to an unknown worker, potentially due to network issues or errors during the interactive communicating job.</entry>
    <entry key="LostAnUnknownWorkerWithCause" context="error">The client lost connection to an unknown worker, potentially due to network issues or errors during the interactive communicating job.\nThis was caused by {0}.</entry>
    <entry key="LostClient" context="error">The worker lost connection to the client, potentially due to network issues or errors during the interactive communicating job.</entry>
    <entry key="LostClientWithCause" context="error">The worker lost connection to the client, potentially due to network issues or errors during the interactive communicating job.\nThis was caused by {0}.</entry>
    <entry key="LostRemoteEndPoint" context="error">Lost connection to remote end point {0}, potentially due to network issues or errors during the interactive communicating job.</entry>
    <entry key="LostWorker" context="error">客户端丢失与工作进程 {0} 的连接，可能是由于网络问题或交互式通信作业过程中的错误。</entry>
    <entry key="LostWorkerWithCause" context="error">The client lost connection to worker {0}, potentially due to network issues or errors during the interactive communicating job.\nThis was caused by {1}.</entry>
    <entry key="OperationTerminatedByUser" context="diagnostic">在并行池启动时，用户终止了操作。池未处于活动状态。</entry>
    <entry key="ParpoolProgressConnectedWorkers" context="diagnostic">已连接到 {1,number,integer} 个并行池工作进程中的 {0,number,integer} 个。</entry>
    <entry key="ParpoolProgressReadyWorkers" context="diagnostic">{1,number,integer} 个并行池工作进程中的 {0,number,integer} 个就绪。</entry>
    <entry key="ParpoolProgressWaitingForRunningJob" context="diagnostic">作业已排队。正在等待 ID 为 {0,number,integer} 的并行池作业启动...</entry>
    <entry key="ParpoolProgressWaitingForWorkers" context="diagnostic">作业正在运行。正在等待并行池工作进程连接...</entry>
    <entry key="PerformingDependencyAnalysis" context="diagnostic">正在分析并向工作进程传输文件... </entry>
    <entry key="PoolArrayInvalidArgument" context="error">Invalid operation on PoolArrayManager.</entry>
    <entry key="PoolDidNotBecomeIdle" context="warning">池在 {0} 毫秒内未变为空闲。某些探查的代码可能仍在运行。</entry>
    <entry key="PoolNotRunning" context="error">并行池当前没有运行。要启动池，请使用 ''parpool''。</entry>
    <entry key="PoolShutDownNoProfile" context="diagnostic">并行池正在关闭。</entry>
    <entry key="PoolShutDownWithProfile" context="diagnostic">使用 ''{0}'' 配置文件的并行池正在关闭。</entry>
    <entry key="PoolShutdown" context="error">此并行池已关闭。</entry>
    <entry key="PoolShutdownNotice" context="warning">使用在线许可的并行池将关闭</entry>
    <entry key="PoolTimeout" context="uistring">已达到 IdleTimeout。</entry>
    <entry key="ReadFromClientError" context="error">A read error occurred while reading from the client.</entry>
    <entry key="ReadFromClientErrorWithCause" context="error">A read error occurred while reading from the client.\nThis was caused by: {0}.</entry>
    <entry key="ReadFromWorkerError" context="error">A read error occurred while reading from worker {0}.</entry>
    <entry key="ReadFromWorkerErrorWithCause" context="error">A read error occurred while reading from worker {0}.\nThis was caused by: {1}.</entry>
    <entry key="RestartOnClusterChangeNotAllowed" context="error">Changing RestartOnClusterChange to true is not allowed for this parallel pool.</entry>
    <entry key="RunOnLabs" context="error">A parallel pool cannot be started from a worker, only from a client MATLAB.</entry>
    <entry key="StartupFailed" context="error">Worker failed to establish communication with the client.</entry>
    <entry key="StartupFailedWithCause" context="error">Worker failed to establish communication with the client.\nThis was caused by: {0}.</entry>
    <entry key="TimeToTimeoutManyMinutes" context="parameter">{0,number,integer} 分钟(剩余 {1,number,integer} 分钟)</entry>
    <entry key="TimeToTimeoutManyMinutesOneRemaining" context="parameter">{0,number,integer} 分钟(剩余不到 1 分钟)</entry>
    <entry key="TimeToTimeoutOneMinute" context="parameter">1 分钟(剩余不到 1 分钟)</entry>
    <entry key="TimeoutConnectingToProfiler" context="error">无法在 {0} 秒内连接到探查器</entry>
    <entry key="TimeoutProcessingProfilerEvents" context="error">无法在 {0} 秒内处理挂起的探查事件</entry>
    <entry key="TimeoutFromPrevious" context="error">在等待 {0} 秒后，并行池中没有可用于远程执行的工作进程。这可能是因为前一个 SPMD 块或 PARFOR 循环无法正确完成而中断，或因为有至少一个工作进程正忙于执行 PARFEVAL。如果此问题仍存在，您可能需要重新启动并行池。</entry>
    <entry key="TimeoutSettingUpSession" context="error">A timeout occurred setting up the parallel pool communication.</entry>
    <entry key="TimeoutSettingUpSessionOnClient" context="error">A timeout occurred setting up the parallel pool communication because no worker activity was detected in {0,number,integer} seconds. Use pctconfig("poolstarttimeout") to change this timeout.</entry>
    <entry key="UnableToCompleteTransfer" context="error">Unable to complete transfer for local file(s) {0}.\nRemote error message is:{1}.</entry>
    <entry key="UnableToCompleteUpdate" context="warning">无法完成本地文件 {0} 的更新。\n远程错误消息为: {1}</entry>
    <entry key="UnableToUpdateWorkerFiles" context="warning">无法更新工作进程文件。\n这是由以下原因造成的: {0}</entry>
    <entry key="UnexpectedParforFailure" context="error">PARFOR 过程中出现意外错误: {0}</entry>
    <entry key="UnknownError" context="error">未知错误。</entry>
    <entry key="UnsupportedFeature" context="error">语法在基于线程的池上不受支持。</entry>
    <entry key="WorkerAborted" context="error">A worker has stopped responding. Delete and restart the parallel pool using parpool.</entry>
    <entry key="WorkerIdentifier" context="parameter">工作进程 {0,number,integer}</entry>
    <entry key="WorkerIdentifierUnknown" context="parameter">丢失工作进程</entry>
    <entry key="WorkerIndexFull" context="parameter">{0,number,integer} (任务 {1,number,integer}；主机: {2})</entry>
    <entry key="WriteToClientError" context="error">A write error occurred while sending to the client.</entry>
    <entry key="WriteToClientErrorWithCause" context="error">A write error occurred while sending to the client.\nThis was caused by: {0}.</entry>
    <entry key="WriteToWorkerError" context="error">A write error occurred while sending to worker {0}.</entry>
    <entry key="WriteToWorkerErrorWithCause" context="error">A write error occurred while sending to worker {0}.\nThis was caused by: {1}.</entry>
    <entry key="RemoteInvocationError" context="error">Unable to invoke method {0} on remote service {1}.</entry>
    <entry key="RemoteInvocationErrorWithCause" context="error">Unable to invoke method {0} on remote service {1}.\nThis was caused by: {2}.</entry>
    <entry key="UnknownRemoteExecutionFailure" context="error">Unknown remote execution failure:\n{0}.</entry>
  </message>
</rsccat>
